analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantToCharArrayCall.CSharp11.cs

```diff
@@ -8,9 +8,9 @@ public class RedundantToCharArrayCall
     {
```

Comment:
It was already there, but as I have spotted lets fix it. The class should not be called `RedundantToCharArrayCall` because it is a redundant ToArrayCall.
Also I suggest to rename the Method from CreateNew2 to something more common. And the propertyValue parameter is not needed either.